changelogs/fragments/69578-shell-remote_tmp-quoting.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - shell - Fix quoting of mkdir command in creation of remote_tmp in order to allow spaces and other special characters.
```

```suggestion
  - shell - fix quoting of mkdir command in creation of remote_tmp in order to allow spaces and other special characters (https://github.com/ansible/ansible/issues/69577).
```
(See https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to for the latest instructions.)

---

Hi @briankohles,

can you update the changelog message.

Thanks

---

Sorry missed the lowercase f, just fixed that now. Hopefully it doesn't mess up the build again. I'll keep an eye on it.